boards.txt

```diff
@@ -1421,6 +1421,20 @@ Disco.menu.pnum.DISCO_F407VG.build.variant=STM32F4xx/F407V(E-G)T_F417V(E-G)T
 Disco.menu.pnum.DISCO_F407VG.openocd.target=stm32f4x
 Disco.menu.pnum.DISCO_F407VG.debug.svd_file={runtime.tools.STM32_SVD.path}/svd/STM32F4xx/STM32F407.svd
 
+# DISCO_F411VE board
+Disco.menu.pnum.DISCO_F411VE=STM32F411E-DISCO
+Disco.menu.pnum.DISCO_F411VE.upload.maximum_size=524288
+Disco.menu.pnum.DISCO_F411VE.upload.maximum_data_size=131072
+Disco.menu.pnum.DISCO_F411VE.build.mcu=cortex-m4
+Disco.menu.pnum.DISCO_F411VE.build.fpu=-mfpu=fpv4-sp-d16
+Disco.menu.pnum.DISCO_F411VE.build.float-abi=-mfloat-abi=hard
```

Comment:
I don't know how to discover the correct configuration for these values.

Reply:
It is the same for all F4, use the one from the genF4:
https://github.com/stm32duino/Arduino_Core_STM32/blob/e96c3a11fb51b1d1e32120555b8863d9febb316c/boards.txt#L4712-L4714